spec/index.bs

```diff
@@ -5879,7 +5881,7 @@ dictionary GPUStorageTextureBindingLayout {
         The access mode for this binding, indicating readability and writability.
 
         Note:
-        There is currently only one access mode, {{GPUStorageTextureAccess/"write-only"}},
+        Currently only one access mode, {{GPUStorageTextureAccess/"write-only"}} is valid,
```

Comment:
If the other two modes are present in the IDL then this needs to be enforced in the validation steps. At the very least the `createBindGroupLayout()` algorithm should be updated to ensure that storage textures must use `"write-only"`. It currently doesn't include any validation for the storage texture access (aside from a probably outdated check that you can't use `"write-only"` on the `VERTEX` stage?)

Reply:
I want to add the API side to this, but have only done WGSL so far. So that is a stop gap and not expected to be merged as is (one of the reasons this PR is a draft).

Reply:
I think I've made the necessary changes, but PTAL.